### ISSUE TITLE
Dark mode

### ISSUE THREAD

**Issue by benoit74** [openzim/devdocs#20]
Do we want to implement a dark mode just like devdocs.io is supporting? (not sure how all readers are passing the appropriate info, maybe worth to PoC if it is complex to implement in scraper)

**Comment by josephlewis42** [comment]
Absolutely! Changing the class on the `<html>` element from `_theme-default` to `_theme-dark` should get us most of the way there:

![image](https://github.com/user-attachments/assets/8df73ebe-3380-485f-ae96-61e810b85e86)

We can do that in JS based on user preferences (and/or a button) but it's likely users will see a flash on each page load before the JS kicks in and switches the page to dark mode.  DevDocs is using Service Workers today so they can avoid that, but I don't think that's an option for us.

**Comment by benoit74** [comment]
Cool ! I think that waiting for JS is the proper move. And if done with Vue.JS (our preference for now, but not a requirement, maybe it is too much work given devdocs specificities) I'm sure we can handle this in a way where there won't be a flickering

Service workers are indeed not an option, they are not supported by too many ZIM readers and a nightmare to support for these readers (we tried with zimit 1 ... we failed)
